controllers/topics.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopicData } = require("../models/topics.models.js");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopicData()
+    .then((topicsData) => {
+      res.status(200).send({ topics: topicsData });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
Good work implementing this controller, great to see you've set up a controller file with the response and catch block implemented. At this stage you have no error handling middleware set up in `app.js` but you will do soon. Sending the response on a key of `topics` is a well names key here as it represents the data being sent back. Also the naming of your controller function `getTopics` is good.